data/deep missions.txt

```diff
@@ -718,36 +720,35 @@ mission "Deep: Remnant Technology"
+		dialog `You recieve a message from Ivan after you land. "Thanks for all your help recently. We brought our findings to the Deep government and they gave us more funding! We've transfered 250,000 credits to your account and made sure you were listed as a primary contributor on all the reports of these immense discoveries. Please meet us on <destination> if you wish to assist us further. Your help would be appreciated, but we will make progress even if you're a bit busy at the moment."`
 	
-	on decline
-		set "Deep: Remnant Research: done" # Allows the player to decline this and any mission beyond this point and still get a chance to unlock the Bactrian, but not doing the other half of the Deep: Remnant chain could ruin future situations for the player.
+	on fail
+		set "Deep: Remnant Research: done" # Allows the player to decline/abort this and any mission beyond this point and still get a chance to unlock the Bactrian, but not doing the other half of the Deep: Remnant chain could ruin future situations for the player.
 
 
 
 mission "Deep: Remnant: Engines"
 	landing
 	name `Fetching Engines`
-	description `Acquire a Forge-Class Thruster and Forge-Class Steering from the Remnant and bring the engines to <destination>.`
+	description `Acquire "Forge-Class" thruster and steering engines from the Remnant and bring them to <destination>.`
 	source Valhalla
 	to offer
 		has "Deep: Remnant Technology: done"
 	on offer
 		conversation
-			`Thanks to the increased funds, the team has moved their research lab to a much more advanced facility which is located closer to the spaceport itself. "This will make transfering material from your ship to the lab much easier, which will be important in these next few missions," Ivan explains.`
-			`	The team of scientists begin explaining to you what they've discovered from the scanner logs you provides. The Remnant, although clearly human, have somehow advanced far beyond what the rest of humanity has achieved.`
+			`Thanks to the new funding grants from the Deep, the team has moved their research lab into a more advanced facility, closer to the spaceport. "This new location will make transfering cargo from your ship to the lab much easier, something we expect to do a lot of in the next few months," Ivan explains. "A careful study of the outfit scans you collected has indicated that the Remnant technology uses unprecedented advancements and improvements that haven't been seen in any human technology before.`
 			
 			branch friendly
-				has "ember waste label"
-			`	"We know that you told us that they tried to attack you, but we are very interested in what their technology may hold. Therefore we need you to be willing to plunder some of their technology and bring it to us to study and analyze."`
+				"reputation: Remnant" >= 1
```

Comment:
The thing about using reputation is that the player could have accepted the blood test but later attacked the Remnant while doing these missions. At that point, it would make no sense for Ivan to say `Even though the Remnant has attacked you` because you never told him they attacked you. You told him they were friendly but later on your decided to attack them.